src/main/rules/GCI1024/IaC/kubernetes/GCI1024.asciidoc

```diff
@@ -0,0 +1,54 @@
+Chaque conteneur défini dans un Pod, un Deployment, ou un StatefulSet Kubernetes doit obligatoirement déclarer une readinessProbe et une livenessProbe.
```

Comment:
the path of this file is wrong ...
please check another example 
ASCIIDOC file will be inside "langugage" directory. Here "kubernetes"